config/api/routes/files.php

```diff
@@ -47,17 +51,28 @@
 			// move_uploaded_file() not working with unit test
 			// @codeCoverageIgnoreStart
 			return $this->upload(function ($source, $filename) use ($path) {
-				$props = [
+				// handle chunked uploads
+				$source = Upload::chunk($this, $source, $filename);
+
+				// if file is not yet complete, return
+				if ($source === null) {
+					return;
```

Comment:
Works already, but wondering if for these non-last chunk responses a different HTTP response code or so should be sent?

Reply:
This one sounds good but is unfortunately depreacted: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/102

206? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/206 

They both seem to only fit partially.

Reply:
I think 206 is for the reverse scenario: where I send a request to receive a range of content and the response only contains that range. So wouldn't touch that. Maybe as is is fine

Reply:
I agree, 200 seems fine.